lib/runner.js

```diff
@@ -124,6 +124,11 @@ exports.execute = function (scripts, options, reporter, callback) {
         return script._root;
     });
 
+    const onlyNodes = scripts.reduce((nodes, script) => nodes.concat(script._onlyNodes), []);
+    if (onlyNodes.length) {
+        internals.skipEverythingExceptOnlyPath(scripts, onlyNodes[0]);
```

Comment:
consider rename, maybe `skipAllButOnly` ... needs to be shorter and simpler: `skipNononly` `filterSkip` `skipExperiments`